.circleci/config.yml

```diff
@@ -4,19 +4,18 @@ version: 2
 # see https://circleci.com/blog/circleci-hacks-reuse-yaml-in-your-circleci-config-with-yaml/
 # and https://learnxinyminutes.com/docs/yaml/
 
-experimental:
-  notify:
-    branches:
-      only:
-        - master
-
 templates:
   job_template: &job_template
     docker:
-      - image: datadog/datadog-agent-runner-circle:latest
+      - image: datadog/datadog-agent-runner-circle:six
```

Comment:
note: should be switched back to `latest` at some point, either before or after merging depending on your plan

Reply:
I'll change `latest` to point to what's `six` now and remove the old Dockerfile in a dedicated PR